addons/l10n_do/data/fiscal_position_template.xml

```diff
@@ -7,52 +7,56 @@
         <!-- Principal Fiscal Position for Dominican Republic internally -->
         <record id="position_buyer_final" model="account.fiscal.position.template">
             <field name="chart_template_id" ref="do_chart_template"/>
-            <field name="name">Consumidor Final</field>
+            <field name="name">Consumidor Final - Obsoleto</field>
         </record>
         <record id="position_buyer_moral" model="account.fiscal.position.template">
             <field name="chart_template_id" ref="do_chart_template"/>
-            <field name="name">Crédito Fiscal</field>
+            <field name="name">Crédito Fiscal - Obsoleto</field>
         </record>
         <record id="position_provider_moral" model="account.fiscal.position.template">
             <field name="chart_template_id" ref="do_chart_template"/>
-            <field name="name">Proveedor Fiscal</field>
+            <field name="name">Proveedor de Servicios</field>
         </record>
         <record id="position_construction_moral" model="account.fiscal.position.template">
             <field name="chart_template_id" ref="do_chart_template"/>
-            <field name="name">Proveedor Fiscal (Construcción)</field>
+            <field name="name">Proveedor de Vigilancia</field>
         </record>
         <record id="position_construction_person" model="account.fiscal.position.template">
             <field name="chart_template_id" ref="do_chart_template"/>
-            <field name="name">Persona Física (Construcción)</field>
+            <field name="name">Persona Física (Construcción) - Obsoleto</field>
         </record>
         <record id="position_person" model="account.fiscal.position.template">
             <field name="chart_template_id" ref="do_chart_template"/>
-            <field name="name">Persona Física</field>
+            <field name="name">Persona Física de Servicios</field>
         </record>
         <record id="position_nonformal" model="account.fiscal.position.template">
             <field name="chart_template_id" ref="do_chart_template"/>
-            <field name="name">Proveedor Informal (Bienes)</field>
+            <field name="name">Proveedor Informal de Bienes</field>
         </record>
         <record id="position_exterior" model="account.fiscal.position.template">
             <field name="chart_template_id" ref="do_chart_template"/>
-            <field name="name">Proveedor del Exterior</field>
+            <field name="name">Servicios del Exterior</field>
         </record>
         <record id="position_expenses" model="account.fiscal.position.template">
             <field name="chart_template_id" ref="do_chart_template"/>
-            <field name="name">Gastos Menores</field>
+            <field name="name">Gastos Menores - Obsoleto</field>
```

Comment:
@smetl This fiscal positions that have Obsoleto = Deprecated

The most sane thing to do (from my POV) is to delete this fiscal positions instead of adding this ugly comment. Is this possible?